SECURITY-INSIGHTS.yml

```
@@ -0,0 +1,52 @@
+header:
+  schema-version: 1.0.0
+  last-updated: '2025-03-12'
+  last-reviewed: '2025-03-12'
+  expiration-date: '2026-03-12T01:00:00.000Z'
+  project-url: https://github.com/open-cluster-management-io/ocm
+  project-release: '0.15.2'
+  changelog: 'https://github.com/open-cluster-management-io/ocm/releases'
+  license: 'https://github.com/open-cluster-management-io/ocm/blob/main/LICENSE'
+project-lifecycle:
+  stage: active
+  bug-fixes-only: false
+  core-maintainers:
+  - 'https://github.com/open-cluster-management-io/community/blob/main/MAINTAINERS.md'
+  roadmap: 'https://github.com/orgs/open-cluster-management-io/projects/2'
+  release-process: 'https://github.com/open-cluster-management-io/community/blob/main/RELEASE.md'
+contribution-policy:
+  accepts-pull-requests: true
+  accepts-automated-pull-requests: true
+  code-of-conduct: https://openssf.org/community/code-of-conduct
+documentation:
+- https://github.com/ossf/security-insights-spec/blob/main/specification.md
```

```suggestion
- https://github.com/ossf/security-insights-spec/tree/main/specification
```

Is it this one? It's currently a dead link.